ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -150,7 +150,6 @@ private Long getIfModifiedSinceMs(RestRequest restRequest) {
   private void setHeadResponseHeaders(BlobInfo blobInfo, RestResponseChannel restResponseChannel)
       throws RestServiceException {
     BlobProperties blobProperties = blobInfo.getBlobProperties();
-    restResponseChannel.setHeader(RestUtils.Headers.LAST_MODIFIED, new Date(blobProperties.getCreationTimeInMs()));
```

Comment:
Why this line is removed?

In addition, not related to this PR, line 81 should be security service is closed.